content/en/serverless/step_functions/installation.md

```diff
@@ -14,10 +14,21 @@ further_reading:
 * Linked Lambda traces are supported for Node.js (layer v112+) and Python (layer v95+) runtimes.
 
 ### How it works
-Datadog AWS Step Functions Monitoring collects logs and integration metrics from the AWS integration and uses ingested logs from AWS Step Functions to generate enhanced metrics and traces for your Step Function executions.
+AWS Step Functions is a fully managed service, and the Datadog Agent cannot be directly installed on Step Functions. However, Datadog can monitor Step Functions through logs.
+
+You can send your Step Functions logs to Datadog by using one of the following methods:
+
+- Use the [Datadog Forwarder][6]. For instructions, see the [Setup](#setup) section on this page.
+- Use Amazon Firehose. For instructions, see [Send AWS service logs to the Datadog Amazon Data Firehose destination][7].
+
+Datadog uses these ingested logs to generate [enhanced metrics][8] for your Step Function executions.
```

Comment:
@cswatt  Here is some context that might be helpful to you.
- Logs are utilized to generate enhanced metrics and traces. 
- Customers can activate enhanced metrics by adding the DD_ENHANCED_METRICS tag to each of their Step Functions. When only enhanced metrics are enabled, customers are only billed for Serverless Workload Monitoring.
- Alternatively, customers can enable serverless tracing, which automatically includes enhanced metrics. With serverless tracing, customers are charged for both the Serverless Workload Monitoring and Serverless APM ([link to the pricing page](https://www.datadoghq.com/pricing/?product=serverless-monitoring#products)), which is calculated based on the count of state transitions. 
- To enable serverless tracing, customers can add the DD_TRACE_ENABLED tag to each Step Function or enable tracing for all Step Functions at the DD Forwarder level (see [this PR](https://github.com/DataDog/datadog-serverless-functions/pull/831)).